addons/api/.eslintrc.js

```diff
@@ -11,17 +11,19 @@ module.exports = {
   parserOptions: {
     ecmaVersion: 'latest',
     sourceType: 'module',
-    // This is specific babel-config. If grows consider creating a babel config file
     requireConfigFile: false,
     babelOptions: {
       plugins: [
         ['@babel/plugin-proposal-decorators', { decoratorsBeforeExport: true }],
       ],
     },
-    // end of babel-config
   },
   plugins: ['ember'],
-  extends: ['eslint:recommended', 'plugin:ember/recommended', 'prettier'],
+  extends: [
+    'eslint:recommended',
+    'plugin:ember/recommended',
+    'plugin:prettier/recommended',
```

Comment:
We probably shouldn't use the `plugin:prettier/recommended` plugin, we want to avoid having prettier rules run as eslint rules and keep them separate. It's kind of confusing but the `prettier` extension is actually from [eslint-config-prettier](https://github.com/prettier/eslint-config-prettier) which does the opposite.

Reply:
Thank you for linking the docs! Good catch, didn't realize this. Ill revert this change.